configs/sst_middleware-tomcat.yaml

```diff
@@ -0,0 +1,22 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Tomcat packages
+  description: Middleware SST - Tomcat packages
+  maintainer: sst_middleware
+
+  packages:
+  - tomcat
+  - tomcat-admin-webapps
+  - tomcat-docs-webapp
+  - tomcat-el-3.0-api
+  - tomcat-jsp-2.3-api
+  - tomcat-lib
+  - tomcat-servlet-4.0-api
+  - tomcat-webapps
+  - mod_proxy_cluster
```

Comment:
mod_proxy_cluster is currently a placeholder in sst_cs_infra_services-httpd; as it was just added to rawhide, it should be a real package now, question is in which workload/SST it belongs.

Reply:
Thanks. Fixed in  f10aa2b - `sst_middleware` owns this package in RHEL 8+,